k8s/postgresql/chart/postgresql/values.yaml

```diff
@@ -0,0 +1,6 @@
+postgresql:
+  image: null
+  volumeSize: null
```

Comment:
Please consider renaming to volumeSizeInGi or similarly to make it easily readable what metric is used.